integration-tests/src/test/java/tachyon/worker/DataServerIntegrationTest.java

```diff
@@ -107,6 +113,9 @@ private void assertValid(final DataServerMessage msg, final int expectedSize,
   @Before
   public final void before() throws IOException {
     System.setProperty(Constants.WORKER_DATA_SERVER, mDataServerClass);
+    if (!mNettyTransferType.equals(UNUSED_TRANSFER_TYPE)) {
+      System.setProperty(Constants.WORKER_NETTY_FILE_TRANSFER_TYPE, mNettyTransferType);
```

Comment:
I think the System property should be cleared in `after()`? On a more stylistic note, does setting or not setting the netty transfer type matter if its unused (ie. is the if necessary)?